vivisect/impapi/windows/advapi_32.py

```diff
@@ -0,0 +1,685 @@
+# APIs for Windows 32-bit advapi library.
```

Comment:
i love all the work you've put into organizing these!

because the standard use for `impapi` is `vivisect.impapi.<OS>.<ARCH>`, what do you think about putting these "real" modules in a subfolder, such as `vivisect/impapi/windows/apis/` or something like that?  so that this folder is still cleanly named after the architecture?